dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/ConnectorArguments.java

```diff
@@ -229,6 +232,37 @@ public class ConnectorArguments extends DefaultArguments {
           .withOptionalArg()
           .describedAs("sekr1t");
 
+  private final OptionSpec<ZonedDateTime> optionStartDate =
+      parser
+          .accepts(
+              OPT_START_DATE,
+              "Inclusive start date for data to export, value will be truncated to hour")
+          .withOptionalArg()
+          .ofType(Date.class)
+          .withValuesConvertedBy(ZonedParser.withDefaultPattern(DayOffset.START_OF_DAY))
+          .describedAs("2001-01-15[ 00:00:00.[000]]");
+
+  private final OptionSpec<ZonedDateTime> optionEndDate =
+      parser
+          .accepts(
+              OPT_END_DATE,
+              "Inclusive end date for data to export, value will be truncated to hour")
+          .withOptionalArg()
+          .ofType(Date.class)
+          .withValuesConvertedBy(ZonedParser.withDefaultPattern(DayOffset.START_OF_DAY))
+          .describedAs("2001-01-15[ 00:00:00.[000]]");
+
+  private final OptionSpec<Integer> optionLookbackDays =
+      parser
+          .accepts(
+              OPT_LOOKBACK_DAYS,
+              "The N days back of data to export from current moment. "
```

Comment:
Please, add more details about `start_date` parameter (it's not always a current moment).

Reply:
There is next line:
```
+ "Can be combined with --"
                  + OPT_START_DATE)
```

Reply:
the description updated to be more clear